### ISSUE TITLE
Angular 2 Template Driven Form radio button Not working.

### ISSUE THREAD

**Issue by MrPardeep** [angular/angular#4084]
There is problem occurred while fetching the value from the radio button in the angular 2 template driven forms. i have used "[(ng-model)]" two way binding of angular 2 for getting the value from the radio button. it does't return neither string nor boolean.

**Comment by PascalPrecht** [comment]
@MrPardeep Please provide a plunk so we have a chance to take a look at your scenario

**Comment by MrPardeep** [comment]
@PascalPrecht Thanks for quick response. i have created plunker with HTML and .ts file.Link is here.

http://plnkr.co/edit/b5plRRPLaDEaLSDBEdxH?p=preview

**Comment by MrPardeep** [comment]
@PascalPrecht i am waiting for your response.

**Comment by PascalPrecht** [comment]
@MrPardeep hey I took a look at your plunk. Unfortunately it's entirely broken. Template paths don't match etc. could you fix that first? Otherwise it's hard to follow and understand your actual problem

**Comment by mhevery** [comment]
stale

**Comment by GregWoods** [comment]
I believe this needs reopening.

Working [plnkr ](http://plnkr.co/edit/46duKk?p=preview)
I had radio button 2 way binding working in Beta1 to Beta3 using a third party radio_value_accessor.ts, but have not succeeded in getting it working in Beta 6 - Beta9

Not sure if we need a fix, or just some guidance, but I'm sick of issue after issue being routinely closed regarding radio buttons, without any apparent understanding of what we are asking here. If 2-way binding of radio buttons in template syntax does work, we at least need a example to show us all how to do it.

**Comment by mhevery** [comment]
@GregWoods please file this as a new issue.

**Comment by GregWoods** [comment]
new issue #7642

**Comment by kibbled** [comment]
any word on why this was closed? Template driven radio buttons still do not work
